src/api/jobApi.ts

```diff
@@ -2,8 +2,8 @@ import axios from "axios"
 import { Job } from "../state/constants/jobConstant";
 
 export const fetchJobsData = async (name: string) => {
-  // const { data } = await axios.get<Job[]>(`${process.env.REACT_APP_DATA_URL}/jobs`);
   const { data } = await axios.get<Job[]>(`https://6201d6f6b8735d00174cb605.mockapi.io/api/job`);
-  const jobs = data.filter((job: Job) => job.title.toLocaleLowerCase().includes(name.toLocaleLowerCase()));
+  const jobs = data.filter((job: Job) => Number(job.id)<=100);
```

Comment:
I think this task's purpose is getting the first 100 jobs, not filtering jobs with id <=100, right

Reply:
- Status : done 
- Note
I use array index instead of id to get first 100 jobs.

Reply:
we can also use Array slice(), but this code is good to me, so resolve it

Reply:
- Fix use array slice().
- status : done.
- please check again for me. thanks